src/main/java/com/google/jenkins/plugins/computeengine/ComputeEngineLinuxLauncher.java

```diff
@@ -146,7 +146,7 @@ private boolean testCommand(ComputeEngineComputer computer, Connection conn, Str
 
     }
 
-    private GoogleKeyPair setupSshKeys(ComputeEngineComputer computer) throws CloudNotFoundException, IOException, InterruptedException {
+    private GoogleKeyPair setupSshKeys(ComputeEngineComputer computer) throws IOException, InterruptedException {
```

Comment:
Same comment as above.

Reply:
Same comment as above..

And this method is not even throwing it itself..